src/clojure_mauth_client/validate.clj

```diff
@@ -16,7 +16,7 @@
         :request_time time
         :client_signature signature}})
 
-(defn- signature-map [msg]
+(defn signature-map [msg]
   (let [signature msg
         values-fn (fn [s] (rest (re-find #"\A(\S+)\s*([^:]+):([^:]+)\z" s)))]
```

Comment:
Today I learned about the `\A` and `\z` escapes. I'll have to start using those in the future – I've been misusing `^` and `$`.